hasher-matcher-actioner/examples/integrations/submit_to_sns_trigger/python/main.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import os
+import json
+import boto3
+
+
+SUBMIT_TOPIC_ARN = os.environ["SUBMIT_TOPIC_ARN"]
+
+
+def lambda_handler(event, context):
+    sns_client = boto3.client("sns")
```

Comment:
Have typically done this in a cached function. I realize this is an example, so can merge as-is too.